runtime/near-runtime-fees/src/lib.rs

```diff
@@ -110,16 +110,10 @@ pub struct AccessKeyCreationConfig {
 /// Describes cost of storage per block
 #[derive(Debug, Serialize, Deserialize, Clone, Hash, PartialEq, Eq)]
 pub struct StorageUsageConfig {
-    /// Base storage usage for an account
-    pub account_cost: Gas,
-    /// Base cost for a k/v record
-    pub data_record_cost: Gas,
-    /// Cost per byte of key
-    pub key_cost_per_byte: Gas,
-    /// Cost per byte of value
-    pub value_cost_per_byte: Gas,
-    /// Cost per byte of contract code
-    pub code_cost_per_byte: Gas,
+    /// Number of bytes for an account record.
+    pub num_bytes_account: u64,
+    /// Additional number of bytes for a k/v record
+    pub num_bytes_data_record: u64,
```

Comment:
I suggest we emphasize in the name that this is "additional", e.g.:

```suggestion
    pub num_extra_bytes_record: u64,
```